src/Common/src/Common.Utils/IO/Steeltoe.Common.IO.csproj

```
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
```

```suggestion
    <Description></Description>
    <PackageTags></PackageTags>
  </PropertyGroup>

  <Import Project="..\..\..\..\versions.props" />
  <Import Project="..\..\..\..\sharedproject.props" />
```

It looks like this will produce a nuget package... at least the Description should be filled out if that's the intent